.golangci.yml

```diff
@@ -0,0 +1,30 @@
+run:
+  timeout: 5m
+  skip-dirs-use-default: true
+  modules-download-mode: readonly
+  allow-parallel-runners: true
+
+output:
+  sort-results: true
+
+linters:
+  disable-all: true
+  enable:
+    - deadcode
+    - gci
+    - godot
+    - unparam
+    - unused
+#    - errcheck
+#    - gosimple
+#    - ineffassign
+#    - staticcheck
+#    - structcheck
+#    - typecheck
+#    - varcheck
```

Comment:
Are these TODOs?

Reply:
Yes, for later. I did not want to go all in, because fixing the issues some of these would flag are outside the scope of this PR, but I also wanted to have a bare minimum of linting code quality tools present.